src/components/HomePage/SearchArea/jobTypes.js

```diff
@@ -0,0 +1,25 @@
+const JOB_TYPES = Object.freeze([
+    {
+        value: "ANY",
+        label: "Any"
+    },
+    {
+        value: "SUMMER_INTERNSHIP",
+        label: "Summer Internship"
+    },
+    {
+        value: "CURRICULAR_INTERNSHIP",
+        label: "Curricular Internship"
+    },
+    {
+        value: "PART_TIME",
+        label: "Part-Time"
+    },
+    {
+        value: "FULL_TIME",
+        label: "Full-Time"
+    },
+
+]);
+
+export default JOB_TYPES;
```

Comment:
Should this be a request to the backend? Can also understand hardcoding, as this will likely never change, but would like for consistency :/

Reply:
Probably should be moved to BE in the future.

Reply:
Maybe have both, for safety? Not sure how fast the BE will move so might be best to keep some logic in the FE for now (I've been more adverse to this in the past than I am now)